modules/ssh

```diff
@@ -24,12 +24,27 @@ cd /etc/ssh
 cp sshd_config sshd_config.$$
 
 # these are mostly supported
+# Sources:
+# - http://people.redhat.com/swells/mea/SECSCAN-FirstRun/sshd_config.htm
+# - http://wp.kjro.se/2013/09/06/hardening-your-ssh-server-opensshd_config
+# - http://kacper.blog.redpill-linpro.com/archives/702
+# - https://www.freebsd.org/cgi/man.cgi?sshd_config(5)
 sed -i \
--e 's/PermitRootLogin *yes.*/PermitRootLogin no/' \
--e 's/UsePrivilegeSeparation *no.*/UsePrivilegeSeparation yes/' \
--e 's/StrictModes *no.*/StrictModes yes/' \
--e 's/IgnoreRhosts *no.*/IgnoreRhosts yes/' \
--e 's/PermitEmptyPasswords *yes.*/PermitEmptyPasswords no/' \
+-e 's/#\?MaxAuthTries *[0-9]*.*/MaxAuthTries 2/' \
+-e 's/#\?PermitRootLogin *\(yes\|no\).*/PermitRootLogin no/' \
+-e 's/#\?UsePrivilegeSeparation *\(yes\|no\|sandbox\).*/UsePrivilegeSeparation sandbox/' \
+-e 's/#\?StrictModes *\(yes\|no\).*/StrictModes yes/' \
+-e 's/#\?IgnoreRhosts *\(yes\|no\).*/IgnoreRhosts yes/' \
+-e 's/#\?PermitEmptyPasswords *\(yes\|no\).*/PermitEmptyPasswords no/' \
+-e 's/#\?ChallengeResponseAuthentication *\(yes\|no\).*/ChallengeResponseAuthentication yes/' \
+-e 's/#\?KerberosAuthentication *\(yes\|no\).*/KerberosAuthentication no/' \
+-e 's/#\?GSSAPIAuthentication *\(yes\|no\).*/GSSAPIAuthentication no/' \
+-e 's/#\?GatewayPorts *\(yes\|no\).*/GatewayPorts no/' \
```

Comment:
Most of the improvements are sane, but sometimes X11 forwarding and gateway ports are used, so setting this as a global policy will get in the way.

Reply:
So if I amend this merge request with a commit that:

1. comments out the lines with `GatewayPorts ` and `X11Forwarding `, 
2. adding a descriptive comment text why, 

the merge request is OK for you?

Reply:
yes, I like the rest.

Reply:
Done. See the new commit.